EIPS/eip-x.md

```diff
@@ -105,68 +109,76 @@ MUST emit the `RequestDeposit` event.
-
-The `assets` that will be received on `redeem` or `withdraw` MAY NOT be equivalent to the current value of `convertToAssets(shares)`, as the price can change between request and execution.
+#### pendingDepositRequest
 
-MUST support a redeem request flow where the shares are transferred from `owner` directly where `owner` is `msg.sender`.
+The amount of requested `assets` in pending state for the `operator` to `deposit` or `mint`.
 
-MUST support a redeem request flow where the shares are transferred from `owner` directly where `msg.sender` has ERC-20 approval over the shares of `owner`.
+MUST NOT include any assets in claimable state for `deposit` or `mint`.
 
-SHOULD check `msg.sender` can spend owner funds using allowance.
+MUST NOT show any variations depending on the caller.
 
-MUST emit the `RequestRedeem` event.
+MUST NOT revert unless due to integer overflow caused by an unreasonably large input.
 
 ```yaml
-- name: requestRedeem
+- name: pendingDepositRequest
   type: function
-  stateMutability: nonpayable
+  stateMutability: view
 
   inputs:
-    - name: shares
-      type: uint256
     - name: operator
       type: address
-    - name: owner
-      type: address
+
+  outputs:
+    - name: assets
+      type: uint256
 ```
 
-#### cancelDepositRequest
+#### requestRedeem
 
-Submits an order to cancel the outstanding deposit request. When the cancel deposit request is fulfilled, `maxRedeem` and `maxWithdraw` will be increased, and `redeem` or `withdraw` from ERC-4626 can be used to receive `assets` that were previously locked for deposit.
+Assumes control of `shares` from `owner` and submits a request for asynchronous `redeem/withdraw`. This places the Request in Pending state, with a corresponding increase in `pendingRedeemRequest` for the amount `shares`. 
 
-MUST emit the `CancelDepositRequest` event.
+MAY support either a locking or a burning mechanism for `shares` depending on the vault implemention. 
 
-```yaml
-- name: cancelDepositRequest
-  type: function
-  stateMutability: nonpayable
-```
+MUST support a redeem Request flow where the `shares` control of shares is taken from `owner` directly where `msg.sender` has EIP-20 approval over the `shares` of `owner`.
 
-#### cancelRedeemRequest
+When the Request is claimable, `maxRedeem` and `maxWithdraw` will be increased, and `redeem` or `withdraw` can be called by `operator` to receive `assets`. A Request MAY transition straight to claimable state but MUST NOT skip the claimable state.
 
-Submits an order to cancel the outstanding redemption request. When the cancel redemption request is fulfilled, `maxDeposit` and `maxMint` will be increased, and `deposit` or `mint` from ERC-4626 can be used to receive `shares` that were previously locked for redemption.
+If a Vault uses a locking mechanism for `shares`, those shares MUST be burned from the vault balance upon claiming the Request.
```

Comment:
```suggestion
If a Vault uses a locking mechanism for `shares`, those shares MUST be burned from the vault balance before or upon claiming the Request.
```

Reply:
Also missing backticks on the second `shares`